settings.gradle

```diff
@@ -1 +1 @@
-include ':app', ':core'
+include ':app', ':core', ':ui:intents'
```

Comment:
"intents" looks like grouping files by file type rather than functionality (and dependencies). This could lead to weakly coupled module. Do you want to have it that way?

Reply:
Hmm, my idea was that the module only contains the "cyclic" references that are needed to start the top-most app level from one of the lower levels - not everything `Intent` related. So I agree that the name is not really a good fit. Would `app-launcher` be more clear? Should it be in the `ui` folder or somewhere else?

> grouping files by file type rather than functionality

Ouch, you are right. That's a thing I don't like about the old structure of AntennaPod and now in #4926 I did it again... Maybe we should discuss a module graph before starting with decoupling. I will prepare a proposal and post it in #4661.

Reply:
> Would `app-launcher` be more clear?

For me, `app-launcher` sounds like starting the application, loading preferences, and initializing the main activity. But in fact it's only a UI forwarder or trigger.

As the code is very trivial we could also think about not to have a `:ui:intents` module at all, but move the code to the corresponding services that use the code. Trivial code can be duplicated, that's usually not a pain in maintenance.

Reply:
The module will grow a bit because the download service can start specific pages of the app. When copying, we would also have to copy the string constants, which could then lead to inconsistencies. So I really want to create a module.

Using something a bit longer, how about `app-launch-intent`? :)